docs/geode-native-docs/remote-querying/remote-queries.html.md.erb

```diff
@@ -30,74 +30,105 @@ Querying and indexing operate only on remote cache server contents.
 
 ### <a id="QueryLanguage"></a>Query language: OQL
 <%=vars.product_name%> provides a SQL-like querying language called OQL that allows you to access data stored in <%=vars.product_name%> regions.
+OQL is very similar to SQL, but OQL allows you to query complex objects, object attributes, and methods.
 
-<%=vars.product_name%> uses a query syntax based on OQL (Object Query Language) to query region
-data. OQL is very similar to SQL, but OQL allows you to query complex objects, object attributes,
-and methods.
+In the context of a query, specify the name of a region by its full path, starting with a forward slash (/ ).
```

Comment:
might "code" tag the forward slash instead of parentheses... make it stand out more